python/ray/tests/test_asyncio.py

```diff
@@ -155,6 +156,8 @@ def throw_error(self):
         await actor.throw_error.remote().as_future()
 
     ray.kill(actor)
+    wait_for_actor_killed(actor._actor_id.hex(),
+                          ray.actors(actor._actor_id.hex())["NumRestarts"])
```

Comment:
Why not call `ray.actors(actor._actor_id.hex())["NumRestarts"]` inside `wait_for_actor_killed`?

Reply:
My bad, we should call` ray.actors(actor._actor_id.hex())["NumRestarts"]` before `ray.kill`.

Reply:
My point is to make the test case simpler. If the pattern is fixed, you can move `ray.kill(actor)` into the helper function as well and rename the helper function to `kill_actor_and_wait_for_failure`.

Reply:
SGTM, thx.